src/components/ProjectList.js

```diff
@@ -66,7 +41,7 @@ class ProjectList extends React.Component {
   }
 
   getProjects(){
-    apiClient.type('projects').get({id: MOBILE_PROJECTS, cards: true, sort: 'display_name'})
+    apiClient.type('projects').get({id: MOBILE_PROJECTS, cards: true, tags: this.props.tag, sort: 'display_name'})
```

Comment:
TBH, I looked up how the Projects/Discipline filter works in PFE, and that is how they query / filter on Disciplines in the project-card-list component.  I'm not sure why it has that name, but I'm guessing that disciplines are "tagged" on the project.